iommu_registers.adoc

```diff
@@ -236,10 +239,18 @@ the IOMMU. At reset, the register shall contain the IOMMU supported features.
 |30    |`HPM`     |RO         | IOMMU implements a hardware performance monitor.
 |31    |`DBG`      |RO        | IOMMU supports the translation-request interface
 |37:32 |`PAS`      |RO        | Physical Address Size (value between 32 and 56)
-|47:38 | reserved  |RO        | Reserved for standard use
-|63:48 |_custom_   |RO        | _Reserved for custom use_
+|38    |`PD20`     |RO        | Three level PDT with 20-bit process_id supported.
+|39    |`PD17`     |RO        | Two level PDT with 17-bit process_id supported.
+|40    |`PD8`      |RO        | One level PDT with 8-bit process_id supported.
+|55:41 | reserved  |RO        | Reserved for standard use
+|63:56 |_custom_   |RO        | _Reserved for custom use_
 |===
 
+The implementation must support a `PSCID` that is at least as wide as the 
+supported `process_id` width. The behavior of configuring a `PSCID` in in-memory
```

Comment:
The implementation must support a `PSCID` that is at least as wide as the widest supported `process_id` in the system.